src/components/entity/ha-entity-picker.ts

```diff
@@ -77,6 +77,22 @@ export class HaEntityPicker extends LitElement {
   @property({ type: Array, attribute: "include-unit-of-measurement" })
   public includeUnitOfMeasurement?: string[];
 
+  /**
+   * Show only these entities
+   * @type {Array}
+   * @attr include-entities
+   */
+  @property({ type: Array, attribute: "include-entities" })
+  public includeEntities?: string[];
+
+  /**
+   * Show only these entities
+   * @type {Array}
+   * @attr exclude-entities
+   */
+  @property({ type: Array, attribute: "exclude-entities" })
+  public excludeEntities?: string[];
```

Comment:
Added to `entity-picker` for performance.

Include made sense so that we didn't iterate every single entity and filter in the ones in the list. It bypasses the rest of the filtering.